src/i18n/rpi-imager_ja.ts

```diff
@@ -0,0 +1,752 @@
+    <message>
+        <location filename="../main.qml" line="150"/>
+        <location filename="../main.qml" line="1038"/>
+        <source>CHOOSE STORAGE</source>
+        <translation>ストレージを選ぶ</translation>
+    </message>
+    <message>
+        <location filename="../main.qml" line="180"/>
+        <source>WRITE</source>
+        <translation>書き込む</translation>
+    </message>
+    <message>
+        <location filename="../main.qml" line="185"/>
+        <source>Select this button to start writing the image</source>
+        <translation>書き込みをスタートさせるにはこのボタンを押してください</translation>
+    </message>
+    <message>
+        <location filename="../main.qml" line="230"/>
+        <source>CANCEL WRITE</source>
+        <translation>書き込みをキャンセル</translation>
+    </message>
+    <message>
+        <location filename="../main.qml" line="233"/>
+        <location filename="../main.qml" line="968"/>
+        <source>Cancelling...</source>
+        <translation>キャンセル中です...</translation>
+    </message>
+    <message>
+        <location filename="../main.qml" line="245"/>
+        <source>CANCEL VERIFY</source>
+        <translation>確認をやめる</translation>
+    </message>
+    <message>
+        <location filename="../main.qml" line="248"/>
+        <location filename="../main.qml" line="991"/>
+        <location filename="../main.qml" line="1057"/>
+        <source>Finalizing...</source>
+        <translation>最終処理をしています...</translation>
+    </message>
+    <message>
+        <location filename="../main.qml" line="268"/>
+        <source>Select this button to access advanced settings</source>
+        <translation>詳細な設定を変更するのならこのボタンを押してください</translation>
+    </message>
+    <message>
+        <location filename="../main.qml" line="284"/>
+        <source>Using custom repository: %1</source>
+        <translation>カスタムレポジトリを使います: %1</translation>
+    </message>
+    <message>
+        <location filename="../main.qml" line="293"/>
+        <source>Keyboard navigation: &lt;tab&gt; navigate to next button &lt;space&gt; press button/select item &lt;arrow up/down&gt; go up/down in lists</source>
+        <translation>キーボードの操作: 次のボタンに移動する→Tabキー  ボタンを押す/選択する→Spaceキー  上に行く/下に行く→矢印キー（上下）</translation>
+    </message>
+    <message>
+        <location filename="../main.qml" line="313"/>
+        <source>Language: </source>
+        <translation>言語: </translation>
+    </message>
+    <message>
+        <location filename="../main.qml" line="336"/>
+        <source>キーボード: </source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../main.qml" line="524"/>
+        <location filename="../main.qml" line="1025"/>
+        <source>削除</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../main.qml" line="525"/>
+        <source>SDカードをFAT32としてフォーマットする</source>
+        <translation type="unfinished"></translation>
+    </message>
```

Comment:
What's happened here? Are the `source` and `translation` accidentally the wrong way around?

Reply:
Oh, I made a mistake.
Fixed at 6a0bd8b.